### PR TITLE
Update periodic bckgrnd article for Chrome 80.

### LINKED ISSUE #1964: Update Periodic Background Sync article for shipping by default.
https://web.dev/periodic-background-sync/

Target publish date: 2019-12-19

 - [ ] Check this box if this is a hard deadline.

### PR DESCRIPTION
**Pull request by jpmedley**
Fixes #1964

### CODE REVIEW

File: src/site/content/en/blog/periodic-background-sync/index.md [hunk 1]
```diff
@@ -69,7 +73,7 @@ Status
 2. Create initial draft of specification
 </td>
 <td markdown="block">
-In Progress
+[In Progress](https://github.com/WICG/BackgroundSync/blob/master/explainers/periodicsync-explainer.md)
```
Comment (mfriesenhahn) [inline src/site/content/en/blog/periodic-background-sync/index.md]:
```suggestion
<a href="https://github.com/WICG/BackgroundSync/blob/master/explainers/periodicsync-explainer.md" rel="noopener">In Progress</a>
```
MD won't parse without extra whitespace, which can be fussy, so switched to HTML.

File: src/site/content/en/blog/periodic-background-sync/index.md [hunk 2]
```diff
@@ -19,18 +20,21 @@ tags:
...
+  progress, follow [Unlocking new capabilities for the
+  web](https://developers.google.com/web/updates/capabilities).
 {% endAside %}
 
 Have you ever been in any of the following situations?
 
 * Riding a train or subway with flaky or no connectivity
-* Being throttled by your carrier after watching too many videos
+* Been throttled by your carrier after watching too many videos
 * Living in a country where bandwidth struggles to keep up with the demand
```
Comment (mfriesenhahn) [inline src/site/content/en/blog/periodic-background-sync/index.md]:
```suggestion
* Living in a area where bandwidth struggles to keep up with the demand
```
Suggested to widen applicability

File: src/site/content/en/blog/periodic-background-sync/index.md [hunk 3]
```diff
@@ -398,43 +395,18 @@ event to use, and to trigger it as many times as you'd like.
...
-          style="max-width: 75%">
-   </figure>
-
-5. Close, and then reopen DevTools.
+## Using the DevTools interface
 
-6. You should now see a **Periodic Background Sync** section within the
-   *Application* panel.
+Starting in Chrome 80, you'll see a **Periodic Background Sync** section in the
+DevTools *Application* panel.
```
Comment (mfriesenhahn) [inline src/site/content/en/blog/periodic-background-sync/index.md]:
```suggestion
DevTools **Application** panel.
```

File: src/site/content/en/blog/periodic-background-sync/index.md [hunk 4]
```diff
@@ -19,18 +20,21 @@ tags:
...
+  - chrome80
 ---
 
 {% Aside %}
-  The Periodic Background Sync API is available as an [**origin trial**](#ot) in Chrome 77 and
-  later. It is part of the [Capabilities project](https://developers.google.com/web/updates/capabilities).
-  This post will be updated as the implementation progresses.
+  Web apps should be able to do anything native apps can. The
+  Capabilities project, of which Periodic Background Sync is only a part, aims
+  to do just that. To learn about other capabilities and to keep up with their
```
[the remark this answers is not in the document]
Comment (jpmedley) [reply src/site/content/en/blog/periodic-background-sync/index.md]: I just forgot to copy the link.

**Comment by jpmedley** [comment]
PTAL

**Comment by mfriesenhahn** [comment]
LGTM!

File: src/site/content/en/blog/periodic-background-sync/index.md [hunk 5]
```diff
@@ -398,43 +396,18 @@ event to use, and to trigger it as many times as you'd like.
 {% endAside %}
 
 <figure class="w-figure  w-figure--center">
-  <img class="w-screenshot" src="3-sw-panel.png" 
+  <img class="w-screenshot" src="3-sw-panel.png"
        alt="The 'Service Workers' section of the Application panel shows a 'Periodic Sync'
             text field and button." style="max-width: 90%">
 </figure>
 
-## Enabling the DevTools interface
```
Comment (kaycebasques) [inline src/site/content/en/blog/periodic-background-sync/index.md]: @jpmedley I'm not seeing the Periodic Background Sync section enabled by default, even in Chrome 81:

image

**Pull request merged**